apps/onboarding-cdc/src/main/resources/application.properties

```diff
@@ -14,4 +14,10 @@ onboarding-cdc.storage.connection-string=${STORAGE_CONNECTION_STRING:UseDevelopm
 
 onboarding-cdc.retry.min-backoff=${ONBOARDING-CDC-RETRY-MIN-BACKOFF:10}
 onboarding-cdc.retry.max-backoff=${ONBOARDING-CDC-RETRY-MAX-BACKOFF:12}
```

Comment:
maybe 12 hours it too long....you could consider to reduce at most 1 hours

Reply:
fix it